code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -0,0 +1,167 @@
+/mob/living/simple_animal/hostile/megafauna/bubblegum
+	name = "bubblegum"
+	desc = "In what passes for a heirarchy among slaughter demons, this one is king."
+	health = 2500
+	maxHealth = 2500
+	attacktext = "rends"
+	attack_sound = 'sound/magic/demon_attack1.ogg'
+	icon_state = "bubblegum"
+	icon_living = "bubblegum"
+	icon_dead = ""
+	friendly = "stares down"
+	icon = 'icons/mob/lavaland/96x96megafauna.dmi'
+	faction = list("mining")
+	weather_immunities = list("lava","ash")
+	speak_emote = list("gurgles")
+	armour_penetration = 40
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	speed = 1
+	move_to_delay = 10
+	ranged = 1
+	flying = 1
+	mob_size = MOB_SIZE_LARGE
+	pixel_x = -32
+	del_on_death = 1
+	aggro_vision_range = 18
+	idle_vision_range = 5
+	loot = list(/obj/structure/closet/crate/necropolis/dragon)
+	var/charging = 0
+	deathmessage = "sinks into a pool of blood, fleeing the battle. You've won, for now... "
+	death_sound = 'sound/magic/enter_blood.ogg'
+
+/obj/item/device/gps/internal/bubblegum
+	icon_state = null
+	gpstag = "Bloody Signal"
+	desc = "You're not quite sure how a signal can be bloody."
+	invisibility = 100
+
+/mob/living/simple_animal/hostile/megafauna/bubblegum/Life()
+	..()
+/*	if(!charging)
+		blood_warp()*/
+	move_to_delay = Clamp(round((health/maxHealth) * 10), 5, 10)
+
+/mob/living/simple_animal/hostile/megafauna/bubblegum/OpenFire()
+	var/anger_modifier = Clamp(((maxHealth - health)/50),0,20)
+	ranged_cooldown = world.time + ranged_cooldown_time
+
+	if(!charging)
+		blood_warp()
+
+	if(prob(25))
+		blood_spray()
+
+	else if(prob(5+anger_modifier/2))
+		slaughterlings()
+	else
+		if(health > maxHealth/2 && !client && !charging)
+			charge()
+		else
+			charge()
+			sleep(10)
+			charge()
+			sleep(10)
+			charge()
+
+
+/mob/living/simple_animal/hostile/megafauna/bubblegum/New()
+	..()
+	for(var/mob/living/simple_animal/hostile/megafauna/bubblegum/B in mob_list)
+		if(B != src)
+			qdel(src) //There can be only one
```

Comment:
This is silly

don't enforce this like this, if anyone is spawning multiple of them then it's either a bug, or an admin running some events.